builder.py

```diff
@@ -130,6 +131,15 @@ def main():
 			# Enlist file paths to be zipped
 			files_to_zip = retrieve_file_paths(dir_top_level)
 			
+			# Create directory to store the resulting zip files in
```

Comment:
Take this outside the for loop that iterates over each problem since this folder contains the zips for all the problems. The best place for this is right after the instantiation of CFService object in the very start of main function.

Reply:
Moved it outside.